### ISSUE TITLE
serialy not working

### ISSUE THREAD

**Issue by jose1711** [kodi-czsk/plugin.video.teevee.sk#3]
hello, serialy part of teevee.sk is being redirected to topserialy.sk for which there's already an addon. will you accept PR to remove serialy link from this addon, keeping only filmy?

**Comment by lubo** [comment]
Of course. Personally, I'm not interested in developing this plugin anymore, but I'll continue maintaining it as long as it'll be necessary. Therefore, feel free to submit as many pull requests as you wish and I'll happily accept them if the quality is acceptable.

**Comment by jose1711** [comment]
fyi: PR in a queue w/o a comment for more than a month now
